### PR TITLE
[CPT-2017] Adding Reuben as a speaker

### PR DESCRIPTION
**Pull request by cobusbernard**
Ready for merging.

### CODE REVIEW

File: content/events/2017-cape-town/speakers.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 +++
 Title = "Speakers"
 Type = "speakers"
-Description = "Speakers for DevOpsDays Cape Town 2017"
+Description = "Speakers for devopsdays Cape Town 2017"
```
Comment (adrianmoisey) [inline content/events/2017-cape-town/speakers.md]: Any reason for this change? We use DevOpsDays everywhere in our marketing etc

File: content/events/2017-cape-town/program/reuben-honigwachs.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "The state of Upspin.io"
+Type = "ignite"
+Speakers = ["reuben-honigwachs"]
++++
+
+Upspin was announced and went open source in February 2017. It is very immature, it's architectural design is discussed controversially, and is very simple in it's current state. It tries to solve a problem many other projects have tried to tackle in the past, albeit a hopeful candidate for "worldFS".
```
Comment (adrianmoisey) [inline content/events/2017-cape-town/program/reuben-honigwachs.md]: Can you replace the two "it's" with "its" ?

File: content/events/2017-cape-town/speakers.md (hunk 1 above)
Comment (cobusbernard) [reply content/events/2017-cape-town/speakers.md]: This kept cropping up when I use the built in helper script. It looks to only be used by the meta tags for the speaker page. Quick grep of the other events show that only Boston and Singapore are capitalised, other 27 are all lowercase.

Comment (adrianmoisey) [reply content/events/2017-cape-town/speakers.md]: Don't forget that it's also used for the opengraph stuff, ie:
image

**Pull request merged**